src/styles/blocks/nav.scss

```
@@ -0,0 +1,48 @@
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+  &__link {
+    position: relative;
+    height: 60px;
+    display: flex;
+    flex-wrap: nowrap;
+    font-weight: 500;
+    font-size: $fs-12;
+    line-height: 14px;
+    text-transform: uppercase;
+    align-items: center;
+    color: $color-black;
```

```suggestion
    color: $color-navy-dark;
```
Nav links color is not black any more